target/linux/mvebu/base-files/etc/config/omnia

```diff
@@ -0,0 +1,41 @@
+
+config omnia 'main'
+	option global_brightness '100'
+
+config led
+	option sysfs 'omnia-led:all'
+	option color '255 255 255'
+	option autonomous '0'
+
+config led
+	option sysfs 'omnia-led:lan0'
+	option autonomous '1'
+
+config led
+	option sysfs 'omnia-led:lan1'
+	option autonomous '1'
+
+config led
+	option sysfs 'omnia-led:lan2'
+	option autonomous '1'
+
+config led
+	option sysfs 'omnia-led:lan3'
+	option autonomous '1'
+
+config led
+	option sysfs 'omnia-led:lan4'
+	option autonomous '1'
+
+config led
+	option sysfs 'omnia-led:pci1'
+	option autonomous '1'
+
+config led
+	option sysfs 'omnia-led:pci2'
+	option autonomous '1'
+
+config led
+	option sysfs 'omnia-led:pci3'
+	option autonomous '1'
+
```

Comment:
For convenience it would be nice to have every led in here, that would ease setting color because one would not have to lookup the exact name.

Reply:
Done.